Content.Shared/Chat/ChatChannelExtensions.cs

```diff
@@ -12,6 +12,7 @@ public static Color TextColor(this ChatChannel channel)
             ChatChannel.OOC => Color.LightSkyBlue,
             ChatChannel.Dead => Color.MediumPurple,
             ChatChannel.Admin => Color.Red,
+            ChatChannel.AdminChat => Color.HotPink,
```

Comment:
These are both probably too neon, could you try `#d03c10` (a more pastel red) and `#b12bc1` (a pastel purple) instead?